docs/administration.platforms.md

```diff
@@ -1,37 +1,70 @@
 ---
 id: administration.platforms
-title: Platform Operation
+title: meshPlatforms
 ---
 
-Operators can perform various maintainance operations for their platforms. For maintenance of Landing Zones, please see
-[Landing Zone Administration](meshcloud.landing-zones.md).
+A key task for Platform Operators is integrating your Cloud Platforms as meshPlatforms into meshStack. This enables
+tenant creation, replication and metering of your Cloud Platforms. The maintenance includes
+general information about the Cloud Platform as well as structuring and configuring them.
 
-## Platform Notifications
+A central part of it also includes the application of security and governance which is covered in the
```

Comment:
The two points you mention about landing zones and tenants would prob. go better in a bullet list. something like:

```
Besides integrating your cloud platform, the following functionality is also highly useful:
* Manage and create landing zones in the [landing zone administration screen](./linkto)
* View all your existing cloud tenants in the [tenant list](./linkto)
```
I think this also bridges better to your point of this functionality becoming a part of the platform control plane in future.